task_router_queue.go

```diff
@@ -0,0 +1,87 @@
+package twilio
+
+import (
+	"context"
+	"net/url"
+)
+
+const taskRouterQueuePathPart = "TaskQueues"
+
+type TaskRouterQueueService struct {
+	client       *Client
+	workspaceSid string
+}
+
+type TaskRouterQueue struct {
+	Sid                     string     `json:"sid"`
+	AccountSid              string     `json:"account_sid"`
+	FriendlyName            string     `json:"friendly_name"`
+	AssignmentActivityName  string     `json:"assignment_activity_name"`
+	AssignmentActivitySid   string     `json:"assignment_activity_sid"`
+	ReservationActivityName string     `json:"reservation_activity_name"`
+	ReservationActivitySid  string     `json:"reservation_activity_sid"`
+	TargetWorkers           string     `json:"target_workers"`
+	TaskOrder               string     `json:"task_order"`
+	DateCreated             TwilioTime `json:"date_created"`
+	DateUpdated             TwilioTime `json:"date_updated"`
+	URL                     string     `json:"url"`
+	WorkspaceSid            string     `json:"workspace_sid"`
+	MaxReservedWorkers      int        `json:"max_reserved_workers"`
+}
+
+type TaskRouterQueuePage struct {
+	Page
+	TaskRouterQueues []*TaskRouterQueue `json:"task_queues"`
```

Comment:
Similarly should be `TaskQueues` to match the API name

Reply:
Agreed, I've made this change. In order to save you time later on how shall we name worker and workflow? Happy to go simply with `Worker` and `Workflow`?

https://www.twilio.com/docs/taskrouter/api/workers
https://www.twilio.com/docs/taskrouter/api/workflows